JobRunner/JobRunner.py

```diff
@@ -142,8 +155,15 @@ def _watch(self, config):
                         self.logger.error("Too many subtasks")
                         self._cancel()
                         return {'error': 'Canceled or unexpected error'}
-                    self._submit(config, req[1], req[2])
+                    if req[2].get('method').startswith('special.'):
```

Comment:
I wonder if its better to put this special flag into a different field instead of shoving it into the method name.

Reply:
This allows us to generate clients which is nice.